main.go

```diff
@@ -454,9 +455,29 @@ func _main(ctx context.Context) error {
 		return nil
 	}
 
+	// WaitGroup for monitoring goroutines
+	var wg sync.WaitGroup
+
+	// ExchangeBot
+	var xcBot *exchanges.ExchangeBot
+	if cfg.EnableExchangeBot {
+		var botCfg exchanges.ExchangeBotConfig
+		if cfg.DisabledExchanges != "" {
+			botCfg.Disabled = strings.Split(cfg.DisabledExchanges, ",")
+		}
+		botCfg.BtcIndex = cfg.ExchangeCurrency
+		xcBot, err = exchanges.NewExchangeBot(&botCfg)
+		if err != nil {
+			log.Errorf("Could not create exchange monitor. Exchange info will be disabled.")
+		} else {
+			wg.Add(1)
+			go xcBot.Start(ctx, &wg)
```

Comment:
I also noticed that if no exchanges with a BTC-USD rate are enabled, the api endpoins report that the xbot is not enabled, but I didn't see any log messages warning that it could not be enabled.

Reply:
The message to look for is 
```
[ERR] DATD: Could not create exchange monitor. Exchange info will be disabled.
```
I'm seeing it on my side, could you verify next time you test?

I'll also make the messaging more clear there.

Reply:
New messaging looks like

```
[ERR] DATD: Could not create exchange monitor. Exchange info will be disabled: No BTC-fiat exchanges were initialized
```
for initialization error and 
```
[INF] DATD: ExchangeBot monitoring coinbase, coindesk, poloniex, binance, bittrex
```
for success.

Reply:
Thanks.  I must have missed it since I was looking for something from the XBOT sub system.